packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -64,23 +62,34 @@ export class AzureClient {
      */
     public async createContainer(
         containerSchema: ContainerSchema,
-    ): Promise<{ container: IFluidContainer; services: AzureContainerServices }> {
+    ): Promise<{
+        container: IFluidContainer;
+        services: AzureContainerServices;
+    }> {
         const loader = this.createLoader(containerSchema);
 
         const container = await loader.createDetachedContainer({
             package: "no-dynamic-package",
             config: {},
         });
 
-        const rootDataObject = await requestFluidObject<RootDataObject>(container, "/");
-
+        const rootDataObject = await requestFluidObject<RootDataObject>(
+            container,
+            "/",
+        );
+        const createNewRequest = createAzureCreateNewRequest(
+            this.props.connection.orderer,
+            this.props.connection.storage,
+            this.props.connection.tenantId,
+        );
         const fluidContainer = new (class extends FluidContainer {
             async attach() {
                 if (this.attachState !== AttachState.Detached) {
-                    throw new Error("Cannot attach container. Container is not in detached state");
+                    throw new Error(
```

Comment:
Is the presence of this check and error due to lack of proper handling / error message in core runtime? Or difference in behavior? Feels like it should not be here.

Reply:
I believe the error should be propagated correctly from the core runtime and the check is redundant, but removal of it should likely be in a separate PR and not in the scope of this one